ocaml/xapi/xapi_host_helpers.ml

```diff
@@ -296,14 +311,23 @@ module InitiatorName = struct
           List.iter (fun ev ->
               match Event_helper.record_of_event ev with
                 | Event_helper.Host (host_ref, Some host_rec) -> begin
-                  let oc = host_rec.API.host_other_config in
-                  let iscsi_iqn = try Some (List.assoc "iscsi_iqn" oc) with _ -> None in
-                  match iscsi_iqn with
-                  | None -> ()
-                  | Some "" -> ()
-                  | Some iqn when iqn <> host_rec.API.host_iscsi_iqn ->
-                      Client.Client.Host.set_iscsi_iqn rpc session_id host_ref (List.assoc "iscsi_iqn" oc)
-                  | _ -> ()
+                    let oc = host_rec.API.host_other_config in
+                    let iscsi_iqn = try Some (List.assoc "iscsi_iqn" oc) with _ -> None in
+                    begin match iscsi_iqn with
+                      | None -> ()
+                      | Some "" -> ()
+                      | Some iqn when iqn <> host_rec.API.host_iscsi_iqn ->
+                        Client.Client.Host.set_iscsi_iqn rpc session_id host_ref iqn
+                      | _ -> ()
+                    end;
+                    (* Accepted values are "true" and "false" *)
+                    let multipathing = try Some (List.assoc "multipathing" oc |> Pervasives.bool_of_string) with _ -> None in
```

Comment:
What if `multipathing` gets removed from `other-config`? We probably want to disable it, but might be tricky to implement because the setter itself temporarily removes it and adds it back.

Reply:
I was thinking the same thing. It depends on how it was used. But it isn't easy indeed, so we shouldn't deal with that case if we don't have to.

Reply:
I checked XenCenter and it sets `multipathing: false` when turning off multipathing, so thats probably OK, and the admin guide says to set it to false too, so we are probably OK, but might be worth adding a comment here saying that if someone deletes the other-config multipathing key we don't do anything, and that could be improved later.

Reply:
Sounds good. As discussed, we shouldn't worry about this for the above reasons, plus the new setter will be the official way to change multipathing, so we don't have to worry too much about this cause it's just for backward compat.

Reply:
Added the comment. We can merge this now if it's ok, and then I can add the check for empty iscsi iqn in another pr.